andela/matterhorn-cfh
# Feature/151168174/jwt user signin

What does this PR do?
It allows user sign in with JWT token instead of sessions

Description of Task to be completed?
It generates JWT token on user sign in and deletes token on sign-out

How should this be manually tested?
Sign in with your login details. If successful, you should see a token in your application storage. On sign-out, the token should be deleted.

  Any background context you want to provide?
User has to sign up first

 What are the relevant pivotal tracker stories?
#151168174 Users should receive a JWT on successful login

 Screenshots (if appropriate)

    Questions:
N/A
